spec/classes/web_spec.rb

```diff
@@ -190,6 +190,14 @@
           it { is_expected.to contain_file('/etc/zabbix/web/zabbix.conf.php').with_content(%r{^\$DB\['SCHEMA'\] = 'zabbix'}) }
         end
 
+        describe 'with parameter: database_double_ieee754' do
+          let :params do
+            super().merge(database_double_ieee754: 'true')
```

Comment:
if we enforce a boolean datatype, we should not pass a string here
```suggestion
            super().merge(database_double_ieee754: true)
```